azure-pipelines.yml

```diff
@@ -2,72 +2,97 @@ trigger:
   branches:
     include:
     - refs/tags/v*
+    
+jobs:
+- job: Build
```

Comment:
As we have only one job, can we omit this paragraph?

Reply:
yes as there is only one job.